pkg/core/blockchain.go

```diff
@@ -259,6 +259,10 @@ func NewBlockchain(s storage.Store, cfg config.ProtocolConfiguration, log *zap.L
 		cfg.NativeUpdateHistories = map[string][]uint32{}
 		log.Info("NativeActivations are not set, using default values")
 	}
+	if cfg.Hardforks == nil {
+		cfg.Hardforks = map[string]uint32{}
+		log.Info("Hardforks are not set, using default value")
```

Comment:
The default is to enable all of them, that's critical.

Reply:
Sure, see the implementation of `IsHardforkEnabled`. If the hardfork is not in the dictionary, then it's enabled. The behaviour matches the C#'s one.